juanevillam/xtrovert
# Cleanup Initial Next.js Project Setup

## Overview  
This PR cleans up the initial Next.js project setup by removing unnecessary files and styles to provide a clean starting point for development.

---

## Key Changes  

### **Fixes**  
- Deleted `next.svg` and `vercel.svg`.  
- Removed unwanted styles from `tailwind.config.ts`.  
- Deleted `favicon.ico`.  
- Cleaned up unwanted styles from `globals.css`.  
- Removed unwanted JSX from `layout.tsx` and `page.tsx`.  

---

## Reasons for Changes  
Removing default files and styles ensures a minimal and clean project structure, making it easier to build and maintain the application.

---

## How to Test  
1. Run the project to ensure it builds and runs without errors.  
2. Verify that the removed files are no longer present in the repository.  
3. Check that `layout.tsx` and `page.tsx` render correctly without the unwanted JSX.
